be2-scala/README.md

```diff
@@ -68,8 +70,38 @@ The project relies on several sbt dependencies (external libraries) :
 
 ---
 
-
-
 ## Coding convention
 
 Our coding guidelines can be found [here](https://docs.scala-lang.org/style/).
+
+## Install java and sbt on linux and Mac
+
+[skdman](https://sdkman.io/) does a pretty good job for installing the required
+environment:
+
+```sh
+# install sdkman
+curl -s "https://get.sdkman.io" | bash 
+# install java zulu 11
+sdk install java 11.0.18-zulu
+# install sbt
+sdk install sbt
+# check that everything went well
+java -version
+> openjdk version "11.0.18" 2023-01-17 LTS
+```
+
+## Build and run
```

Comment:
```suggestion
## Package and run
```

I reckon for day-to-day development, we should probably instead point to `sbt run`